package.json

```
@@ -115,9 +117,12 @@
 		"generate-client:etherpad": "node ./scripts/generate-client.js -u 'http://localhost:9001/api/openapi.json' -p 'apps/server/src/infra/etherpad-client/etherpad-api-client' -c 'openapitools-config.json'",
 		"pregenerate-client:tsp-api": "rimraf ./apps/server/src/infra/tsp-client/generated",
 		"generate-client:tsp-api": "openapi-generator-cli generate -c ./openapitools.json --generator-key tsp-api",
+		"pregenerate-client:files-storage-api": "rimraf ./apps/server/src/infra/files-storage-client/generated",
+		"generate-client:files-storage-api": "openapi-generator-cli generate -c ./openapitools.json --generator-key files-storage-api",
+		"pregenerate-client:lessons-api": "rimraf ./apps/server/src/modules/common-cartridge/common-cartridge-client/lesson-client/new-lesson-api-client",
```

remove 'new' from the name
```suggestion
		"pregenerate-client:lessons-api": "rimraf ./apps/server/src/modules/common-cartridge/common-cartridge-client/lesson-client/lesson-api-client",
```